pkgs/by-name/mn/mnamer/package.nix

```diff
@@ -1,31 +1,40 @@
-{ python3Packages, fetchFromGitHub, lib }:
+{
+  python3Packages,
+  fetchFromGitHub,
+  lib,
+}:
 
 python3Packages.buildPythonApplication rec {
   pname = "mnamer";
-  version = "2.5.4";
+  version = "2.5.5";
+  format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "jkwill87";
     repo = "mnamer";
     rev = version;
-    sha256 = "sha256-fONQq/RboWHFuEFU7HP1ThUpSjOIlkg54c2WlMUKwuk=";
+    sha256 = "sha256-qQu5V1GOsbrR00HOrot6TTAkc3KRasBPDEU7ZojUBio=";
   };
 
+  build-system = with python3Packages; [
+    setuptools
+    setuptools-scm
+  ];
+
   propagatedBuildInputs = with python3Packages; [
+    appdirs
     babelfish
+    guessit
     requests
-    appdirs
-    teletype
     requests-cache
-    guessit
+    teletype
   ];
 
-  patches = [
-    # requires specific old versions of dependencies which have been updated in nixpkgs
-    ./remove_requirements.patch
+  pythonRelaxDeps = true;
```

Comment:
I tried specifying the packages that needed this, but even when I specified `requests-cache` and `setuptools-scm` it failed here:

```
Executing pythonRuntimeDepsCheck
Checking runtime dependencies for mnamer-2.5.5-py3-none-any.whl
  - requests-cache~=0.9.7 not satisfied by version 1.2.1
  - setuptools-scm~=7.1.0 not satisfied by version 8.1.0
```

Reply:
Did you specify as `pythonRelaxDeps = [ "requests-cache" "setuptools-scm" ]`, or differently?

Reply:
Like that, yes.

Reply:
That's odd. In that case your solution works